### PR TITLE
Attempt to optimize serialization performance

### PR DESCRIPTION
**Pull request by wu-clan**
About https://github.com/fastapi-practices/fastapi_best_architecture/discussions/265

### CODE REVIEW

File: backend/app/common/response/response_schema.py [hunk 1]
```diff
@@ -53,60 +52,41 @@ class ResponseBase:
...
-        :param kwargs: jsonable_encoder 配置项
         :return:
         """
-        if data is not None:
-            # TODO: custom_encoder 配置失效: https://github.com/tiangolo/fastapi/discussions/10252
-            custom_encoder = {datetime: lambda x: x.strftime(settings.DATETIME_FORMAT)}
-            kwargs.update({'custom_encoder': custom_encoder})
-            data = jsonable_encoder(data, exclude=exclude, **kwargs)
-        return {'code': res.code, 'msg': res.msg, 'data': data}
+        return ResponseModel(code=res.code, msg=res.msg, data=data)
```
Comment (downdawn) [inline backend/app/common/response/response_schema.py:73]: Should **kwargs be retained?

**Review by downdawn — APPROVED** [review]

**Comment by downdawn** [comment]
LGTM

File: backend/app/common/response/response_schema.py (hunk 1 above)
Comment (wu-clan) [reply backend/app/common/response/response_schema.py:73]: **kwargs is no longer relevant here.

File: backend/app/api/v1/user.py [hunk 2]
```diff
@@ -48,10 +48,10 @@ async def password_reset(request: Request, obj: ResetPassword):
     return await response_base.fail()
 
 
-@router.get('/me', summary='获取当前用户信息', dependencies=[DependsJwtAuth])
+@router.get('/me', summary='获取当前用户信息', dependencies=[DependsJwtAuth], response_model_exclude={'password'})
```
Comment (wu-clan) [inline backend/app/api/v1/user.py:51]: The alternative is to use the original parameters in the interface.

**Pull request merged**